src/lib/characterPreview/ShowcasePortrait.tsx

```diff
@@ -20,22 +20,24 @@ export function ShowcasePortraitLarge() {
 }
 
 export function ShowcasePortrait(props: {
-  source: ShowcaseSource,
-  character: Character,
-  displayDimensions: ShowcaseDisplayDimensions,
+  source: ShowcaseSource
+  character: Character
+  displayDimensions: ShowcaseDisplayDimensions
   customPortrait: CustomImageConfig | undefined
   editPortraitModalOpen: boolean
   setEditPortraitModalOpen: (b: boolean) => void
   onEditPortraitOk: (p: CustomImagePayload) => void
   simScoringResult: SimulationScore
-  artistName: string | undefined,
-  setOriginalCharacterModalInitialCharacter: (c: Character) => void,
-  setOriginalCharacterModalOpen: (b: boolean) => void,
-  setCharacterModalAdd: (b: boolean) => void,
-  onPortraitLoad: (img: string) => void,
+  artistName: string | undefined
+  setOriginalCharacterModalInitialCharacter: (c: Character) => void
+  setOriginalCharacterModalOpen: (b: boolean) => void
+  setCharacterModalAdd: (b: boolean) => void
+  onPortraitLoad: (img: string) => void
 }) {
   const { t } = useTranslation(['charactersTab', 'modals', 'common'])
   const globalThemeConfig = window.store((s) => s.globalThemeConfig)
+  const showcaseUID = window.store((s) => s.savedSession.showcaseUID)
+  const UID = window.store((s) => s.scorerId)
```

Comment:
These two seemsredundant, couldn't we just use the existing scorerId and not have showcaseUID at all?

Reply:
Oh I see, its the boolean toggle, ignore me